docs/accounts/Basics.md

```diff
@@ -31,6 +31,23 @@ Each of these will have their own requirements for opening, which will be checke
 <!-- theme: warning -->
 > SIPPs are currently in a private beta. If you wish to access them, please contact your account manager for more details
 
+## Account currencies
+
+Each account created within the system can have an allocated currency. The currencies currently supported are GBP, EUR and USD. Once the currency of an account has been set, it cannot be changed. Any [Portfolios](docs/guides/adc4b06b3d4ed-portfolio-basics) created under an account will be opened in the same currency as the account itself. 
+
+Account currency can only be set on GIAs. The currency does not need to be set for other account types when they are created. 
```

Comment:
We should be pushing tenants to always set the currency, regardless of account type. There are restrictions as you've mentioned, but the behaviour where we default to GBP is intended to be temporary.

My suggestion would be:
```suggestion
Account currency should be set for all account types, however some account types have restrictions on which currencies can be used. See below for a summary:

| Account type | Allowed currencies |
|--------------|--------------------|
| GIA          | GBP, USD, EUR      |
| ISA          | GBP                |
| JISA         | GBP                |
| SIPP         | GBP                |
```